packages/webdriverio/src/utils/implicitWait.ts

```diff
@@ -1,15 +1,22 @@
 import logger from '@wdio/logger'
 const log = logger('webdriverio')
 
+export type CurrentElement = {
```

Comment:
Can we create an issue for this to clean this up later?

Reply:
What's the idea, move this to some declaration file?

Reply:
the problem is that we generate the interface properties for the webdriverio element to attach its methods compiled from the protocol json files .. not exactly sure how this should look like atm.

Reply:
This is not fully clear to me yet, do you want to create the issue? Or I could create a draft of it to help! 🙂